src/components/TagInput/TagInput.jsx

```diff
@@ -0,0 +1,53 @@
+import { useState } from "react";
+import closeIcon from "./../../assets/icon/close-icon.svg";
+import styles from "./TagInput.module.css";
+
+const TagInput = ({ tags, onChange }) => {
+  const [input, setInput] = useState("");
+
+  const handleKeyDown = (e) => {
+    if (e.key === "Enter" || e.key === " ") {
+      e.preventDefault();
+      const trimmed = input.trim();
+      if (trimmed && !tags.includes(trimmed)) {
+        onChange([...tags, trimmed]);
+      }
+      setInput("");
+    }
+  };
+
+  const handleRemove = (tagToRemove) => {
+    const nextTags = tags.filter((tag) => tag !== tagToRemove);
+    onChange(nextTags);
+  };
+  return (
+    <div className={styles.tagInput}>
+      <label className={styles.label}>태그</label>
+      <input
+        type="text"
+        value={input}
+        placeholder="태그를 입력해주세요"
+        onChange={(e) => setInput(e.target.value)}
+        onKeyDown={handleKeyDown}
+        className={styles.input}
+      />
```

Comment:
💊 제안
라벨과 인풋을 사용하셨으니 연결해서 라벨 클릭시 자동으로 인풋으로 포커스가 가게 해주세요.
```suggestion
      <label htmleFor="id" className={styles.label}>태그</label>
      <input
        id="id"
        type="text"
        value={input}
        placeholder="태그를 입력해주세요"
        onChange={(e) => setInput(e.target.value)}
        onKeyDown={handleKeyDown}
        className={styles.input}
      />
```